gateway/src/types.ts

```diff
@@ -74,9 +74,12 @@ export interface ProviderProxy {
   // TODO(Marcelo): Remove now - this should live in the routingGroups.
   priority?: number
 
-  /** Weather to disable the key in case of error, if missing defaults to True. */
+  /** Whether to disable the key in case of error, if missing defaults to True. */
   disableKey?: boolean
 
+  /** Whether to block the key in case of error, if missing defaults to True. */
```

Comment:
does it really default to true? if so then maybe my earlier suggestion was wrong